app/src/main/java/org/simple/clinic/shortcodesearchresult/ShortCodeSearchResultState.kt

```diff
@@ -1,28 +1,27 @@
 package org.simple.clinic.shortcodesearchresult
 
-import org.simple.clinic.allpatientsinfacility.PatientSearchResultUiState
-import org.simple.clinic.patient.PatientSearchResult
 import org.simple.clinic.plumbing.AsyncOp
 import org.simple.clinic.plumbing.AsyncOp.IN_FLIGHT
 import org.simple.clinic.plumbing.AsyncOp.SUCCEEDED
+import org.simple.clinic.searchresultsview.PatientSearchResults
 
 data class ShortCodeSearchResultState(
     val shortCode: String,
     val fetchPatientsAsyncOp: AsyncOp,
-    val patients: List<PatientSearchResultUiState>
+    val patients: PatientSearchResults
 ) {
   companion object {
     fun fetchingPatients(
         bpPassportNumber: String
     ): ShortCodeSearchResultState =
-        ShortCodeSearchResultState(bpPassportNumber, IN_FLIGHT, emptyList())
+        ShortCodeSearchResultState(bpPassportNumber, IN_FLIGHT, PatientSearchResults.emptyResults())
```

Comment:
Can we make this a constant property instead of a function invocation?

`PatientSearchResults.EMPTY_RESULTS`